tests/integration/mounts.bats

```diff
@@ -19,3 +19,12 @@ function teardown() {
 	[ "$status" -eq 0 ]
 	[[ "${lines[0]}" == *'/tmp/bind/config.json'* ]]
 }
+
+@test "runc run [ro tmpfs mount]" {
+	update_config 	' .mounts += [{"source": "tmpfs", "destination": "/mnt", "type": "tmpfs", "options": ["ro", "nodev", "nosuid", "mode=755"]}] 
+			| .process.args |= ["grep", "/mnt", "/proc/mounts"]' 
```

Comment:
Slightly more correct check would be something like
```suggestion
			| .process.args |= ["grep", "^tmpfs /mnt ", "/proc/mounts"]' 
```

Reply:
done